sentry_streams/sentry_streams/deployment_config/alerts_config.yaml

```diff
@@ -0,0 +1,37 @@
+env:
+  topics:
+    logical-events: events
+    transformed-events: transformed-events
+    transformed-events-2: transformed-events-2
+
+# These are intended to be runtime-agnostic config
+pipeline:
+  sources_config:
+    myinput:
+      bootstrap_servers: localhost:9092
+      consumer_group: python-test
+      auto_offset_reset: latest
+
+  sinks_config:
+    kafkasink:
+      bootstrap_servers: localhost:9092
+
+  reduce_config:
+    myreduce:
+      parallelism: 3
+
+flink:
+  parallelism: 2
+  sources_config:
+    myinput:
```

Comment:
In https://github.com/getsentry/streams/pull/79 I abstracted away the type of source/sink. So we should pick the right source type in configuration. So you should have a representation of KafkaSource (with kafka paramters) and opening up the possibility to have other types.